helm/node-exporter-app/templates/daemonset.yaml

```diff
@@ -25,7 +25,7 @@ spec:
       securityContext:
         runAsUser: {{ .Values.userID }}
         runAsGroup: {{ .Values.userGroup }}
-      priorityClassName: giantswarm-critical
+      priorityClassName: system-node-critical
```

Comment:
Same issue as before, this also runs on CP in monitoring NS

Reply:
Are you 100% sure? I don't think so. what it runs in the CP is https://github.com/giantswarm/g8s-prometheus/tree/master/helm/g8s-prometheus-chart/templates/node-exporter

Reply:
Not certain, but it would be better to check with team magic to check back. Looks like you are right but we should be running our apps in CP as well IMO.

Reply:
Yep we should but as @rossf7 said in our discussion this morning we are not there yet https://gigantic.slack.com/archives/C02MS174K/p1578482740025100?thread_ts=1578480073.018600&cid=C02MS174K

@giantswarm/team-magic any advice here?

Reply:
We don't run this node-exporter on CPs. g8s-prometheus holds the chart for control plane node exporters.

Reply:
Yes currently this chart is only used in tenant clusters. Since system-node-critical is a build in priority class the e2e test is OK. 

So no objections from an apps POV. 👍